### PR TITLE
Feature error tips

### PR DESCRIPTION
**Pull request by rylandg**
Initial PR to get feedback on the exact wording. Brought about because of this old issue.

https://github.com/binaris/binaris/issues/166

Tests failure seems unrelated.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -47,6 +47,10 @@ const exceptionWrapper = function tryCatchWrapper(funcToWrap) {
       process.exit(0);
     } catch (err) {
       logger.error(err.message);
+      if (process.env.BINARIS_LOG_LEVEL === 'info' || !process.env.BINARIS_LOG_LEVEL) {
```
Comment (arielshaqed) [inline lib/index.js]: This is odd.  E.g. it logs if I set `BINARIS_LOG_LEVEL=info`, but _not_ if i set `BINARIS_LOG_LEVEL=warn` or `BINARIS_LOG_LEVEL=quux`.

You definitely want to log if `!process.env.BINARIS_LOG_LEVEL`.  And arguably you might also want to log this if `!logger.isDebugEnabled` (or somesuch; also this works in Winston v3 I think but I dunno about v2).

File: lib/index.js [hunk 2]
```diff
@@ -47,6 +47,10 @@ const exceptionWrapper = function tryCatchWrapper(funcToWrap) {
...
+        // eslint-disable-next-line max-len
+        logger.warn('\nTip: You can export BINARIS_LOG_LEVEL=[silly|debug|verbose|info|warn|error] to view debug logs');
```
Comment (arielshaqed) [inline lib/index.js]: Is it really true e.g. for `error`?

Comment (rylandg) [reply lib/index.js]: I mean those are the logging levels we have defined. Of course error level works.

```
const colorLevels = {
  silly: 'white',
  debug: 'white',
  verbose: 'white',
  info: 'white',
  warn: 'yellow',
  error: 'red',
};
```

File: lib/index.js (hunk 1 above)
Comment (rylandg) [reply lib/index.js]: I don't understand. It's not disabling logging. It's just showing you a tip about logging levels if the user doesn't have a log level defined or if INFO is set.

File: lib/index.js (hunk 2 above)
Comment (arielshaqed) [reply lib/index.js]: Again, the code is easy to understand (yay!).  But I disagree with what it does.  The context in which I see the hint message is that I just had an error message.  And this is when you tell me I could "export `BINARIS_LOG_LEVEL=...error... to view debug logs`".  It's a good time to suggest `verbose` or `debug` or `silly` (even then, _only_ those that will actually give extra information!), but it's hardly a good time to tell me about levels that are more restrictive than I already have.

File: lib/index.js (hunk 1 above)
Comment (rylandg) [reply lib/index.js]: Ok that's more direct feedback. I don't mind removing the `info` case.

File: lib/index.js (hunk 2 above)
Comment (rylandg) [reply lib/index.js]: Understood.

**Comment by rylandg** [comment]
@arielshaqed your turn again

File: test/cases.yml [hunk 3]
```diff
@@ -10,13 +10,19 @@
   steps:
     -   in: bn deploy -p /home/dockeruser/test/trollycopter icydiceymolecule
         err: "ENOENT: no such file or directory, open '/home/dockeruser/test/trollycopter/binaris.yml'"
+        out: |-
+            
+            Tip: You can export BINARIS_LOG_LEVEL=[silly|debug|verbose] to view debug logs
```
Comment (arielshaqed) [inline test/cases.yml]: Consider not repeating this each time, and just putting a `*` in most of the outputs.

**Pull request merged**